front/app/containers/Admin/projects/project/description/index.tsx

```diff
@@ -155,21 +155,20 @@ const ProjectDescription = memo<
             <SectionField>
               {!moduleActive && (
                 <QuillMultilocWithLocaleSwitcher
-                  id="project-description"
+                  id="project-description-module-inactive"
                   valueMultiloc={formValues.description_multiloc}
                   onChange={handleDescriptionOnChange}
                   label={formatMessage(messages.descriptionLabel)}
                   labelTooltipText={formatMessage(messages.descriptionTooltip)}
                   withCTAButton
                 />
               )}
-              <Outlet
-                id="app.containers.Admin.projects.edit.description.projectDescriptionBuilder"
-                onMount={setModuleToActive}
+              <ProjectDescriptionBuilderToggle
                 valueMultiloc={formValues.description_multiloc}
                 onChange={handleDescriptionOnChange}
                 label={formatMessage(messages.descriptionLabel)}
                 labelTooltipText={formatMessage(messages.descriptionTooltip)}
+                onMount={setModuleToActive}
```

Comment:
Let's remove this legacy `onMount` and `setModuleToActive` setup, and replace the `moduleActive` variable with just the `project_description_builder` feature flag?